src/domain/event/eventInfo/__tests__/EventInfo.test.tsx

```diff
@@ -206,3 +210,45 @@ it('should create ics file succesfully when end time is not defined', async () =
 
   expect(FileSaver.saveAs).toBeCalled();
 });
+
+it('should show audience age info in signle course page', async () => {
```

Comment:
Very well coveraged tests! 👍

Reply:
Perhaps one case can be added for the test:
`return ageLimit !== '0+' ? `${ageLimit} -${t('event.info.age')}` : '';`
should not render the text. Perhaps checking that age title is not present.

Reply:
true that. i ll add

Reply:
fixed